src/librustc_metadata/cstore_impl.rs

```diff
@@ -120,11 +120,6 @@ impl CrateStore for cstore::CStore {
         self.get_crate_data(krate)
     }
 
-    fn describe_def(&self, def: DefId) -> Option<Def> {
```

Comment:
You didn't move this to the macro invocation at the top of this file so there's no way for it to actually do anything right now.